console/console-init/ui/mock-console-server/package.json

```diff
@@ -19,6 +19,7 @@
   "homepage": "https://github.com/k-wall/mock-console-server#readme",
   "dependencies": {
     "apollo-server": "^2.9.5",
+    "clone": "^2.1.2",
```

Comment:
I don't see this actually being used?

Reply:
Actually absence of this is breaking the mock server. As clone module is present in the mock-console-server.js file. link- https://github.com/riccardo-forina/enmasse/blob/master/console/console-init/ui/mock-console-server/mock-console-server.js (line no 10)

Reply:
Oh it’s in the server, I didn’t notice that. Thanks